src/proto/ctl/storage_nvme.proto

```diff
@@ -60,8 +60,8 @@ message NvmeController {
 	string pciaddr = 3;	// pci address
 	string fwrev = 4;	// firmware revision
 	int32 socketid = 5;	// NUMA socket ID
-	repeated Namespace namespaces = 6;	// controller's namespaces
-	repeated Health healthstats = 7;	// controller's health stats
+	Health healthstats = 6;	// controller's health stats
+	repeated Namespace namespaces = 7;	// controller's namespaces
```

Comment:
Not an issue just yet, but I'm wondering at what point we should start limiting the Protobuf communication changes. I.e. not re-using field numbers from obsoleted fields.

Reply:
don#t see it as a particularly big issue at the moment but I guess that#s a post-1.0 stabilisation phase